config/provider-metadata.yaml

```diff
@@ -11039,7 +11399,7 @@ resources:
         importStatements: []
     aws_cloudfront_cache_policy:
         subCategory: CloudFront
-        description: Provides a cache policy for a CloudFront ditribution. When it’s attached to a cache behavior, the cache policy determines the values that CloudFront includes in the cache key. These values can include HTTP headers, cookies, and URL query strings. CloudFront uses the cache key to find an object in its cache that it can return to the viewer. It also determines the default, minimum, and maximum time to live (TTL) values that you want objects to stay in the CloudFront cache.
+        description: Use the
```

Comment:
This doesn't seem right. Maybe the parser in the code generator ran into something it couldn't handle?